adage/analyze/tests.py

```diff
@@ -1140,5 +1155,26 @@ def testSearchPA14(self):
             'E-GEOD-24262'
         )
 
+    def testSampleIndex(self):
+        """
+        Make sure that analyze.search_indexes.SampleIndex.prepare_experiments
+        produced the expected result. We should be able to find each sample we
+        indexed in setUp() by searching on the corresponding Experiment's
+        accession number.
+        """
+        respEGEOD24262 = self.api_client.get(
+            self.searchURI,
```

Comment:
Oh, interesting - this is definitely outside the scope of this pull request, but I forgot that we have one big search endpoint, instead of one search endpoint for each resource, or model (something like `'/api/v0/sample/search/'`). Do you think there is a better approach out of the two, or is it dependent on the project...?

Reply:
I am a fan of having one master search index that covers anything you might wish to find. If needed, filters can be applied for context-specific searches.

We don't quite do this right now because `django-genes` is a separate component, but so far don't have a use case that calls for a search that encompasses `Sample`s, `Experiment`s, *and* `Gene`s either.

Reply:
Ok, gotcha - yep, that makes sense with `django-genes`